docs/app/Examples/elements/Container/Variations/ContainerFluidExample.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable max-len */
+
+import React, { Component } from 'react'
+import { Container, Message } from 'stardust'
+
+export default class ContainerFluidExample extends Component {
+  render() {
+    return (
+      <div>
+        <Message className='info'>
+          Fluid containers are useful for setting text alignment, or other variations on unstyled content
+        </Message>
```

Comment:
Same comment with the `<Message />` here as above.  Let's hoist it up out of the example code.